draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -621,8 +621,14 @@ Other specifications or profiles of this specification may define alternative fo
 
 Security considerations in this section help achieve the following properties:
 
-* Claim values issued by the Issuer cannot be obtained by the adversary, if the Holder did not intend to disclose them.
-* Claim values that are not issued by the Issuer cannot be successfully verified by the Verifier, if the Holder forged the Disclosures.
+**Selective Disclosure:** An adversary in the role of the Verifier cannot obtain
+information from an SD-JWT about any claim name or claim value that was not
+explicitly disclosed by the Holder unless that information can be derived from
+other disclosed claims or external sources.
```

Comment:
For example, if `nationalities` is selectively disclosable, and the holder does not disclose it to the verifier, the verifier might still learn the nationality from 

 - a passport image acquired out of band, e.g., during a video identification session (external source), or
 - a disclosed claim containing the passport number, which may leak information about the nationality ("other disclosed claims").

So we have to say "the verifier does not learn the `nationality` claim's value from the presentation of that claim" but it still may learn it from somewhere else because we cannot control that.